akka-javasdk/src/main/java/akka/javasdk/agent/ConversationHistory.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk.agent;
+
+import java.util.List;
+
+public record ConversationHistory(int maxSize, List<ConversationMessage> messages) { }
```

Comment:
I don't think we should include maxSize in this. Might not make sense for other implementations.